app/services/idv/flows/inherited_proofing_flow.rb

```diff
@@ -3,17 +3,30 @@ module Flows
     class InheritedProofingFlow < Flow::BaseFlow
       STEPS = {
         get_started: Idv::Steps::InheritedProofing::GetStartedStep,
+        agreement: Idv::Steps::InheritedProofing::AgreementStep,
+        verify_info: Idv::Steps::InheritedProofing::VerifyInfoStep,
+        verify_phone: Idv::Steps::InheritedProofing::PhoneStep,
```

Comment:
Not sure I understand having the verify phone step here if we will be redirecting to the phone step when we are wrapped here

Reply:
Maybe just rename the step to reflect what the user is actually doing... ConfirmPhoneUse ??

Reply:
This is a place holder, We will update this as part of the coming sprint('s) when we work on handing off to the phone step.